### PR TITLE
Maintenance

### LINKED ISSUE #5: Improve documentation

### PR DESCRIPTION
**Pull request by maasencioh**
closes #5

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,25 +6,26 @@
...
-
-## [API Documentation](https://mljs.github.io/preprocess/)
-
-## Example
+## Usage
 
 ```js
-const preprocess = require('ml-preprocess');
-```
+import library from 'ml-preprocess';
```
Comment (targos) [inline README.md]: you should update this with a small example

File: src/index.js [hunk 2]
```diff
@@ -1,7 +1,3 @@
-'use strict';
-
-exports.normal = require('./pre/normal');
-exports.scale = require('./pre/scale');
-exports.center = require('./pre/center');
-// exports.select = require('./pre/select');
-// exports.discret = require('./pre/discret');
+export {normal} from './preprocess/normal';
+export * from './preprocess/scale';
```
Comment (targos) [inline src/index.js:2]: this is a breaking change. don't forget to bump the major for the release

File: src/preprocess/normal.js [hunk 3]
```diff
@@ -0,0 +1,62 @@
+import {matrix, array} from 'ml-stat';
```
Comment (targos) [inline src/preprocess/normal.js:1]: matrix is unused

File: src/preprocess/normal.js [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+            } else {
+                ans[i][j] = (data[i][j] - means[i]) / std[i];
+            }
+        }
+    }
+
+    return ans;
+}
+
+function matrixStandardDeviation(data, means) {
```
Comment (targos) [inline src/preprocess/normal.js:56]: is there something wrong with `matrix.standardDeviation`?

File: src/preprocess/normal.js (hunk 3 above)
Comment (maasencioh) [reply src/preprocess/normal.js:1]: It's used on `let means = matrix.mean(data, 1);`

File: src/preprocess/normal.js (hunk 4 above)
Comment (maasencioh) [reply src/preprocess/normal.js:56]: yes, it was returning a `NaN` for some values and I can't select the order as in the median

File: src/preprocess/normal.js (hunk 3 above)
Comment (targos) [reply src/preprocess/normal.js:1]: didn't see it, sorry

File: src/index.js (hunk 2 above)
Comment (maasencioh) [reply src/index.js:2]: thanks :+1:

File: src/preprocess/normal.js (hunk 4 above)
Comment (targos) [reply src/preprocess/normal.js:56]: I prefer that you fix it instead of reimplementing your version

Comment (maasencioh) [reply src/preprocess/normal.js:56]: And I will! just wanted to try the test and advance a bit in this project

**Pull request merged**